apps/rewards/app/components/App/App.js

```diff
@@ -121,62 +181,96 @@ class App extends React.Component {
     this.closePanel()
   }
 
-  yourReward = () => {
+  myReward = reward => {
     this.setState({
-      panel: PANELS.YourReward,
+      panel: PANELS.MyReward,
       panelProps: {
         onClaimReward: this.onClaimReward,
         onClosePanel: this.closePanel,
+        vaultBalance: '432.9 ETH',
```

Comment:
Is this a stub to be dynamic later?

Reply:
Yea, it is, shall be addressed shortly (not right now though).